### DISCUSSION TITLE
Is Vaex active?

### DISCUSSION THREAD

**Discussion by grafail** [vaexio/vaex#2363] [General]
It seems like there has not been any changes to Vaex since December. Is its development still active? I am asking as I still see one of the devs (@maartenbreddels) replying to some issues, but it does not feel certain.

Thanks!

**Comment by erwanp** [comment]
Hello ; I'm updating the question. 

Vaex is certainly a great package and helped us increase the performances of our state-of-the-art scientific radiation code (https://github.com/radis/radis) by orders of magnitude in the past few years. It was a blast!

However, the lack of support of recent Python versions (https://github.com/vaexio/vaex/issues/2397) is impairing the development of our own code; and we must need to have an idea of the long-term support of Vaex before knowing whether our infrastructure should be dependant on it. 

As my understanding from LinkedIn, the main developers @maartenbreddels  and @JovanVeljanoski  have respectively started a new project and secured a full-time job in another company , and probably cannot dedicate much time to Vaex anymore. 
This is very fair (everyone needs to eat and open-source users won't feed you)  ; but if this is confirmed @maartenbreddels  and @JovanVeljanoski ; let us vaex-users know of the long-term status of the package.   

And if Vaex is indeed not going to be be actively developed in the future
1. That's a loss! 
2. I reiterate congratulations for all the work achieved so far and what Vaex has brought to us
3. What would be your next best choice as a memory-mapping DataFrame library ?

**Reply by JovanVeljanoski** [reply]
Hi @erwanp (et al)
I am truly sorry for any frustration the silence and inactivity in this project may have caused. I am aware that you indeed make good use of vaex, and this must be annoying. 

I am however no longer involved in this project, and I am unaware of its status, so unfortunately I can not provide any info or other support. I also think it is not nice to discuss other libraries / alternatives on this forum - but if you see any value in it, feel free to reach out to me privately elsewhere and I'd be happy to share how I approach this in my job for example. 

I am sorry I can't be more helpful at this time - and many thanks for the kind words and support of vaex you and the [radis](https://github.com/radis/radis) team have expressed over the years.

**Comment by maartenbreddels** [comment]
I'll do my best to at least get vaex releases out.
